index.md

```diff
@@ -63,6 +63,10 @@ Python interface into the RACF management application programming interface.
 
 [Download & Install From GitHub](https://github.com/ambitus/pyracf/releases)
 
+&nbsp;
+
+[Check for & set up RACF Authorizations](../common/check_for_and_setup_RACF_authorizations)

```

Comment:
```markdown
[IRRSMO00 Precheck Setup](../common/setup_precheck)
```

Also, for this installation section, maybe we should make this a bulleted or numbered list of todos for when installing pyRACF.

```markdown
1. [Download & Install From GitHub](https://github.com/ambitus/pyracf/releases)
2. [IRRSMO00 Precheck Setup](../common/setup_precheck)
```

Reply:
I like that a lot, and changed it in that way.